src/schemas/components/SchemaFormPropertyInput.vue

```diff
@@ -93,6 +93,8 @@
     }
 
     if (isSchemaPropertyType(type, undefined)) {
+      // automatically switch to prefect_kind === 'json'?
+      // value might not be json, so will need to convert it to json...
       throw 'not implemented'
```

Comment:
In what scenarios would we need to convert the value? If it's coming from the db it should already be serialized so we shouldn't need do anything here I _think_